src/main/java/com/softserveinc/ita/vsaroz/pageobjects/RozetkaPage.java

```diff
@@ -0,0 +1,23 @@
+package com.softserveinc.ita.vsaroz.pageobjects;
+
+import com.codeborne.selenide.SelenideElement;
+
+import static com.codeborne.selenide.Condition.*;
+import static com.codeborne.selenide.Selenide.$x;
+
+public class RozetkaPage {
+
+    private final SelenideElement lapTops = $x("//div[@class='menu-wrapper menu-wrapper_state_static ng-star-inserted']//li[1]");
+    private final SelenideElement filterByLapTop = $x("//a[@title='Ноутбуки']");
+    private final SelenideElement dell = $x("//div[@class='sidebar-block ng-star-inserted']//li[4]");
+
+    public void clickOnMenuItem() { lapTops.click(); }
+
+    public void clickLapTop() { filterByLapTop.click(); }
+
+    public void filterByBrand(String name) {
+        dell
+                .shouldHave(text("Dell"))
```

Comment:
you are passing here the name, why it's not used here? why did you hardcoding names in page objects? it's not flexible and not useful

Reply:
if you create a flexible locator template, you will be able to pass here any laptop producer and it will be chosed